### PR TITLE
Fix cutadapt paired collection output ftype

### PR DESCRIPTION
**Pull request by TomHarrop**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)
 
----

A Galaxy Australia user reported that cutadapt outputs plain text fastq but assigns fastqsanger.gz ftype, if the input is a gzipped paired collection.

It happens because cutadapt writes directly to the `.dat` file if the output is a paired collection, so it doesn't detect the `gz` extension and writes plain text. I think this fixes it by writing to the out1 and out2 files and picking them up in the output `<collection>`.

### CODE REVIEW

File: tools/cutadapt/cutadapt.xml [hunk 1]
```diff
@@ -478,9 +475,17 @@ $read_mod_options.zero_cap
                     </conditional>
                 </repeat>
             </section>
-            <output_collection name="out_pairs" type="paired">
-                <element name="forward" decompress="True" file="cutadapt_out1.fq.gz" ftype="fastq.gz" />
-                <element name="reverse" decompress="True" file="cutadapt_out2.fq.gz" ftype="fastq.gz" />
+            <output_collection name="out_pairs" type="paired" count="2">
+                <element name="forward" decompress="True" file="cutadapt_out1.fq.gz" ftype="fastq.gz">
```
Comment (bernt-matthias) [inline tools/cutadapt/cutadapt.xml:479]: Can you use lower case `true`?

**Review by bernt-matthias — APPROVED** [review]
Looks good to me.

File: tools/cutadapt/cutadapt.xml (hunk 1 above)
Comment (TomHarrop) [reply tools/cutadapt/cutadapt.xml]: Thanks, I've changed those two to `true`. Or did you mean all the `true`s in the XML?

**Comment by TomHarrop** [comment]
> So funny that we discover the bug at the same time...

@lldelisle and in consecutive PRs. What are the chances?

Thanks for the feedback and review.

File: tools/cutadapt/cutadapt.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/cutadapt/cutadapt.xml]: Change all if you like.

**Comment by lldelisle** [comment]
Can you merge like this or you prefer to wait for the `True` -> `true`?

**Comment by bernt-matthias** [comment]
Lets just merge like this :) Thanks a lot for fixing this.

**Comment by lldelisle** [comment]
Thanks @bernt-matthias (I had data waiting to be analyzed...). Thanks @TomHarrop for the PR.

**Pull request merged**